src/ui/nodewidget.cc

```diff
@@ -44,6 +44,8 @@
 namespace veles {
 namespace ui {
 
+const float k_minimap_selection_factor = 10;
```

Comment:
We'll refactor this old convention in few days (after fixing Jenkins), so insert new code in Google C++ Style